scripts/eloot.lic

```diff
@@ -125,12 +125,13 @@ module ELoot
       default_hash = YAML.load_file("#{filename}")
     else
       default_hash = {
-        :loot_types=>["alchemy", "armor", "box", "clothing", "collectible", "gem", "jewelry", "lockpick", "magic", "reagent", "scroll", "skin", "uncommon", "valuable", "wand"],
+        :loot_types=>["alchemy", "armor", "box", "clothing", "gem", "jewelry", "lockpick", "magic", "reagent", "scroll", "skin", "uncommon", "valuable", "wand"],
```

Comment:
Do we really need to take it out of loot types? Taking out of sell_loot_types makes sense, but wouldn't this mess up actually looting of items as well?

Reply:
Your right, good catch.